app/forms/hyrax/composition_form.rb

```diff
@@ -4,23 +4,22 @@ module Hyrax
   # Generated form for Composition
   class CompositionForm < Hyrax::Forms::WorkForm
     self.model_class = ::Composition
-    self.terms -= [:creator, :keyword, :subject]
-    self.terms += [:instruments, :date, :tags,
-                   :length, :genre, :software, :medium, :composer]
+    self.terms += [:duration, :medium, :resource_type,
+                   :technical_specs, :program_notes]
 
-    self.required_fields = [:title, :instruments]
+    self.required_fields = [:title, :subject]
```

Comment:
Consider adding `:creator` to the `required_fields`.

Reply:
Will update it.

Reply:
Just a note: I see now why creator/composer wasn't in the required fields before. The creator/composer was hidden in the previous version, so it didn't matter if it was marked required or not.